config/configmap/inferenceservice.yaml

```diff
@@ -185,3 +185,14 @@ data:
         "cpuRequest": "100m",
         "cpuLimit": "1"
     }
+  defaults: |-
```

Comment:
I think `predictorResourceDefaults` and `transformerResourceDefaults` may be better? I think these two components will have very different typical resource requirements.

Reply:
That makes sense, but in that case may be there should be also `explainersResourceDefaults` ?